### ISSUE TITLE
Colored whitespaces trimmed

### ISSUE THREAD

**Issue by KapJI** [noborus/ov#650]
When using delta colored lines at the end are trimmed.

## less

```
[delta]
side-by-side = true
pager = "less -F"
```

<img width="1151" alt="Screenshot 2024-11-05 at 19 07 56" src="https://github.com/user-attachments/assets/ff891375-6e23-4a8a-abb3-ce1c33b6582a">

## ov

```
[delta]
side-by-side = true
pager = "ov -F"
```

<img width="1150" alt="Screenshot 2024-11-05 at 19 08 20" src="https://github.com/user-attachments/assets/5f62dbc3-25d3-4d0c-95f5-67d37e1ace5f">

Is this a bug? Or can this be configured?

**Comment by noborus** [comment]
Thank you for sending issue.
The cause is that the `ov` has not yet supported the Erase in Line of Ansi Control Sequences.
I'm going to add that support.

**Comment by KapJI** [comment]
Fixed by #651
